tsfile/src/main/java/org/apache/iotdb/tsfile/read/controller/MetadataQuerierByFileImpl.java

```diff
@@ -172,4 +208,19 @@ public void loadChunkMetaDatas(List<Path> paths) throws IOException {
     return chunkMetaDataList;
   }
 
+  private boolean checkAccess(ChunkGroupMetaData chunkGroupMetaData) {
+    if (!partitionMode) {
+      return true; // always true
+    }
+    long startOffsetOfChunkGroup = chunkGroupMetaData.getStartOffsetOfChunkGroup();
+    long endOffsetOfChunkGroup = chunkGroupMetaData.getEndOffsetOfChunkGroup();
+    long middleOffsetOfChunkGroup = (startOffsetOfChunkGroup + endOffsetOfChunkGroup) / 2;
+    if (partitionStartOffset < middleOffsetOfChunkGroup
+        && middleOffsetOfChunkGroup <= partitionEndOffset) {
```

Comment:
Please check whether the partitionEndOffset and partitionStartOffset of two consecutive partitions are equal.

Reply:
![image](https://user-images.githubusercontent.com/7240743/53281727-6c067600-3768-11e9-956f-b0349cf1793b.png)


![image](https://user-images.githubusercontent.com/7240743/53281717-5ee98700-3768-11e9-9a7a-eace765c78fc.png)
![image](https://user-images.githubusercontent.com/7240743/53281720-61e47780-3768-11e9-9450-542eae0f4fa9.png)
![image](https://user-images.githubusercontent.com/7240743/53281723-66109500-3768-11e9-93ac-e85a202f4a32.png)
![image](https://user-images.githubusercontent.com/7240743/53281726-6872ef00-3768-11e9-9c69-775d237f1437.png)